lib/utils/error-handler.js

```diff
@@ -36,7 +36,7 @@ process.on('timing', function (name, value) {
 process.on('exit', function (code) {
   process.emit('timeEnd', 'npm')
   log.disableProgress()
-  if (npm.config.loaded && npm.config.get('timing')) {
```

Comment:
I think we could dry this up by adding something like this near the top, no?

```js
const config = npm.config || { loaded: false }
```

And then replacing `npm.config` with `config` in the lines below.

Reply:
Seems reasonable to me.

Reply:
Hmm, this approach seems to introduce more stack traces and more errors. Is this what we want?

```
$ ./bin/npx-cli.js foo
Command failed: node.exe npm-cli.js config get cache --parseable
Error: Exit prior to config file resolving.
    at errorHandler (...\lib\utils\error-handler.js:149:16)
    at ...\bin\npm-cli.js:150:20
    at get (...\lib\config.js:171:3)
    at EventEmitter.config (...\lib\config.js:68:14)
    at Object.commandCache.(anonymous function) (...\lib\npm.js:156:13)
    at EventEmitter.<anonymous> (...\bin\npm-cli.js:131:30)
    at process._tickCallback (internal/process/next_tick.js:61:11)
npm ERR! Exit prior to config file resolving.
Error: npm.load() required
    at Object.get (...\lib\npm.js:59:13)
    at errorHandler (...\lib\utils\error-handler.js:207:14)
    at ...\bin\npm-cli.js:150:20
    at get (...\lib\config.js:171:3)
    at EventEmitter.config (...\lib\config.js:68:14)
    at Object.commandCache.(anonymous function) (...\lib\npm.js:156:13)
    at EventEmitter.<anonymous> (...\bin\npm-cli.js:131:30)
    at process._tickCallback (internal/process/next_tick.js:61:11)
npm ERR! npm.load() required
...\lib\npm.js:59
      throw new Error('npm.load() required')
      ^

Error: npm.load() required
    at Object.get (...\lib\npm.js:59:13)
    at process.errorHandler (...\lib\utils\error-handler.js:207:14)
    at process.emit (events.js:198:13)
    at process._fatalException (internal/bootstrap/node.js:496:27)
```